crafting_combinator/script/rc.lua

```diff
@@ -140,13 +155,37 @@ function _M:find_ingredients_and_products(forced)
 end
 
 
+function _M:find_machines(forced)
+	local recipe, input_count = recipe_selector.get_recipe(self.entity, nil, defines.circuit_connector_id.combinator_input)
+
+	if self.recipe ~= recipe or forced then
+		self.recipe = recipe
+		local params = {}
+		if recipe and recipe.category then
+			for i, mac in pairs(global.rc.machines[recipe.category] or {}) do
+				local mac_res = self.entity.force.recipes[mac]
+				if mac_res and mac_res.enabled then
+					table.insert(params, {
+						signal = recipe_selector.get_signal(mac),
```

Comment:
This should use different logic than the recipes. Again, it should probably be the item that the machine is built with.

Reply:
It probably does not matter if we output the signal of the item used to place to machine or the recipe you get the item from. I am outputing the item now.